### PR TITLE
Remove local nodePort registry in favor of changed bits service

### PR DESCRIPTION
**Pull request by andreas-kupries**
## Description

Removed `loadbalanced` property.
Moved `nodePort` property from `kubecf` section to `bits` section.

:warning: TODO: Update the bits chart.

## Motivation and Context

See #792
See SUSE/bits-service-release/pull/7 for the bits release changes.

## How Has This Been Tested?

Local minikube, SA eirini. Smoke tests run manually.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: deploy/helm/kubecf/values.yaml [hunk 1]
```diff
@@ -511,7 +507,7 @@ bits:
   kube:
     external_ips: []
   services:
-    loadbalanced: true
+    nodePort: 31666
```
Comment (mook-as) [inline deploy/helm/kubecf/values.yaml:510]: Should we not have a doc comment to mark this hidden?

**Comment by andreas-kupries** [comment]
> I assume we've already updated the subchart via one of the other PRs.

Note the ":warning: TODO: Update the bits chart." in the description.
This is waiting on the corresponding bit release PR to be reviewed, accepted, and merged.

File: deploy/helm/kubecf/values.yaml (hunk 1 above)
Comment (andreas-kupries) [reply deploy/helm/kubecf/values.yaml:510]: I was unsure how the tooling handles the sub-chart configuration.
Will fix.

Comment (andreas-kupries) [reply deploy/helm/kubecf/values.yaml:511]: Done in commit 56f87f06fdedf93929d43f196af36084bc48be5f

**Comment by viovanov** [comment]
Based on @mook-as 's comment in https://github.com/SUSE/bits-service-release/pull/7 it sounds like we still need some Eirini changes?

**Comment by andreas-kupries** [comment]
@gaktive Ping. Given the below I would like to ask some other dev to have a look over this.

# Actions

Rebased this PR to master (bd8ff1e060797fa0fe411453b47fd3ba71313e30) to resolve conflict in `deploy/helm/kubecf/values.yaml`.

# Problem

Re-testing in a local minikube showed this failing several times (eirini smoke), no success (1). It was checked and confirmed that on master the eirini-smoke was :heavy_check_mark: (after the initial re-trials which populate the various image caches / bit storage so that their download/use is fast enough to get under the system timeouts).

# Results

A few retrials with partial PRs were done.
This PR here contains 3 chunks of changes.

  1. Switch from `bits.services.loadbalanced: true` to `bits.services.nodePort: 31666`
  2. Switch from `eirini.env.DOMAIN: '127.0.0.1.nip.io:31666" #'` to `eirini.env.DOMAIN: '127.0.0.1.nip.io'`
  3. Remove `features.eirini.registry.service.nodeport: 31666`.

Treating these as independent changes we have 2**3 = 8 combinations, from none to all.

|1|2|3|Notes|State|
|---|---|---|---|---|
||||master|:heavy_check_mark:|
|||y||:boom:|
||y|||:boom:|
||y|y|Out of time Friday||
|y|||#1182|:heavy_check_mark:|
|y||y|#1185|:heavy_check_mark:|
|y|y|||:boom:|
|y|y|y|Self|:boom:|

Note that the #1182 draft PR had only a single success of the tests in a sea of about 10 failures. For all the others, (1) implies that despite the run of failures we cannot be fully sure if there is a true problem with the combination,  or just the limited resources of my machine borking things up.

Similar for the 1+3 combination (#1185, superset of #1182) I was about to pull the plug and mark it as failed, and then it succeeded. So, it might be that only the combinations containing (2) are bad, and (3) alone.

----
  1. Problem with the local testing of eirini smoke is about 7 minutes for a run, on failure it does a second, so usually 14 to 20 minutes turnaround. This limits the number of tests which can be made in case the issue is racing/resources (minikube had 20G to accomodate the larger eirini, even so it is quite squeezed). Another limit comes from the time needed to re-deploy (even with running foundation) when switching to a different set of changes to trial (~30 min. Yes, I am feeling quite frustrated).

**Comment by gaktive** [comment]
Will need someone to rerun this test matrix to see what's going on with some scenarios.

**Comment by andreas-kupries** [comment]
:warning: This PR and its ticket (#792) will be superseded by any work on #1179.
:warning: Ditto for the derived draft PRs #1182 and #1185 

Consider this blocked.

**Comment by jandubois** [comment]
Superseded by #1179.

**Pull request closed without merging**